### PR TITLE
Allow PHP 8.3

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

**Comment by fezfez** [comment]
waiting https://github.com/weierophinney/laminas-ci-swoole-builder/pull/4

File: psalm-baseline.xml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.12.0@f90118cdeacd0088e7215e64c0c99ceca819e176">
+<files psalm-version="5.15.0@5c774aca4746caf3d239d9c8cadb9f882ca29352">
   <file src="src/AbstractStaticResourceHandlerFactory.php">
+    <ArgumentTypeCoercion>
+      <code>$lastModifiedDirectives</code>
```
Comment (fezfez) [inline psalm-baseline.xml:5]: i haven't find a way to suppress this error, if anyone (like @Ocramius :D ) can check this

Comment (Ocramius) [reply psalm-baseline.xml:5]: OK to baseline this, IMO

**Comment by weierophinney** [comment]
@fezfez Before I tag laminas-ci-swoole-builder... what versions of swoole and openswoole should I build into the container images?

**Comment by fezfez** [comment]
> @fezfez Before I tag laminas-ci-swoole-builder... what versions of swoole and openswoole should I build into the container images?

I don't know but, if the build as it Is ok, I think it's ok to push

**Comment by weierophinney** [comment]
> I don't know but, if the build as it Is ok, I think it's ok to push

That's not how it works. :smile: 

When I tag, I build Swoole and OpenSwoole for each supported PHP version, package them, and push them up to the release as release assets. Those packages are what the CI uses. 

That's why I need to know which versions of Swoole and OpenSwoole to build.

**Comment by fezfez** [comment]
> > I don't know but, if the build as it Is ok, I think it's ok to push
> 
> That's not how it works. 😄
> 
> When I tag, I build Swoole and OpenSwoole for each supported PHP version, package them, and push them up to the release as release assets. Those packages are what the CI uses.
> 
> That's why I need to know which versions of Swoole and OpenSwoole to build.

According to https://github.com/remicollet/remirepo/issues/237 the release are not yet available 

Edit 1 : see also https://github.com/openswoole/ext-openswoole/issues/336

**Comment by weierophinney** [comment]
@fezfez  —

> According to [remicollet/remirepo#237](https://github.com/remicollet/remirepo/issues/237) the release are not yet available

We build from source. So the question is: are there PHP 8.3-compatible versions of swoole and/or openswoole for me to build?

**Comment by fezfez** [comment]
> @fezfez —
> 
> > According to [remicollet/remirepo#237](https://github.com/remicollet/remirepo/issues/237) the release are not yet available
> 
> We build from source. So the question is: are there PHP 8.3-compatible versions of swoole and/or openswoole for me to build?

Like I say, if you look at the link there is no compatible version

**Comment by weierophinney** [comment]
@fezfez It looks like patches exist for each of Swoole and OpenSwoole, but no releases have been made yet. I _may_ be able to accommodate those, but not before next week. As such, this patch is on hold until we can get builds created, which will either be based on the patches that allow each extension to build against 8.3, or after releases are made.

**Comment by gsteel** [comment]
OpenSwoole just released: https://github.com/openswoole/ext-openswoole/releases/tag/v22.1.0

**Comment by fezfez** [comment]
Swoole have been released https://github.com/swoole/swoole-src/releases/tag/v5.1.1

**Comment by Xerkus** [comment]
we are aware

**Comment by Xerkus** [comment]
Picked phpunit fixes for #125
Thank you.

Superseded by #124

**Pull request closed without merging**